code/modules/surgery/organs/organ_internal.dm

```diff
@@ -239,6 +239,95 @@
 	S.reagents.add_reagent("salbutamol", 5)
 	return S
 
+/obj/item/organ/internal/tongue
+	name = "tongue"
+	desc = "A fleshy muscle mostly used for lying."
+	icon_state = "lungs"
+	zone = "head"
+	slot = "tongue"
+	var/say_mod = null
+
+/obj/item/organ/internal/tongue/proc/TongueSpeech(var/message)
+	return message
+
+/obj/item/organ/internal/tongue/Insert(mob/living/carbon/M, special = 0)
+	..()
+	if(say_mod && M.dna && M.dna.species)
+		M.dna.species.say_mod = say_mod
+
+/obj/item/organ/internal/tongue/Remove(mob/living/carbon/M, special = 0)
+	..()
+	if(say_mod && M.dna && M.dna.species)
+		M.dna.species.say_mod = initial(M.dna.species.say_mod)
+
+/obj/item/organ/internal/tongue/lizard
+	name = "forked tongue"
+	desc = "A thin and long muscle typically found in reptilian races, apparently moonlights as a nose."
+	icon_state = "lungs"
+	say_mod = "hisses"
+
+/obj/item/organ/internal/tongue/lizard/TongueSpeech(var/message)
+	var/regex/lizard_hiss = new("s+", "g")
+	var/regex/lizard_hiSS = new("S+", "g")
+	if(copytext(message, 1, 2) != "*")
+		message = lizard_hiss.Replace(message, "sss")
+		message = lizard_hiSS.Replace(message, "SSS")
+	return message
+
+/obj/item/organ/internal/tongue/fly
+	name = "proboscis"
+	desc = "A freakish looking meat tube that apparently can take in liquids."
+	icon_state = "lungs"
+	say_mod = "buzzes"
+
+/obj/item/organ/internal/tongue/fly/TongueSpeech(var/message)
+	var/regex/fly_buzz = new("z+", "g")
+	var/regex/fly_buZZ = new("Z+", "g")
+	if(copytext(message, 1, 2) != "*")
+		message = fly_buzz.Replace(message, "zzz")
+		message = fly_buZZ.Replace(message, "ZZZ")
+	return message
+
+/obj/item/organ/internal/tongue/abductor
+	name = "superlingual matrix"
+	desc = "A mysterious structure that allows for instantaneous and omnipresence communication between users. Pretty impressive until you need to eat something."
+	icon_state = "lungs"
+	say_mod = "gibbers"
+
+/obj/item/organ/internal/tongue/abductor/TongueSpeech(var/message)
+	//Hacks
+	var/mob/living/carbon/human/user = usr
+	var/rendered = "<i><font color=#800080><b>[user.name]:</b> [message]</font></i>"
+	for(var/mob/living/carbon/human/H in mob_list)
+		var/obj/item/organ/internal/tongue/T = H.getorganslot("tongue")
+		if(!T || T.type != type)
```

Comment:
You need to check the 'team' of the tongue somehow, so that multiple ships of abductors can't hear eachother. Though I suppose that's kind of a feature.